erpnext/healthcare/doctype/patient_encounter/patient_encounter.json

```diff
@@ -324,11 +325,16 @@
    "no_copy": 1,
    "print_hide": 1,
    "read_only": 1
+  },
+  {
+   "fieldname": "get_applicable_treatment_plans",
+   "fieldtype": "Button",
+   "label": "Get Applicable Treatment Plans"
```

Comment:
Let's show the button only after Patient selection:

![image](https://user-images.githubusercontent.com/24353136/130765686-e4f1ddd4-f537-4780-a342-4edd133ec1fd.png)